File: Python-code/t1.py
class solution:
    def isp(self,x:int) -> bool:
        s = str(x)
        print(s)

        s1 = s[::-1]
        print(s1)

        if s1==s:
            return True
        else:
            return False

s = solution()
a= s.isp(-121)
print(a)

File: C++code/unit/t1.cpp
/*
1. 两数之和
这道题暴力解法一个个的相加判断

*/

// 如下为我leetcode提交代码

class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> n;
        for(int i=0;i<nums.size()-1;i++)
        {
                for(int j=i+1;j<nums.size();j++)
                {
                    if (nums[i]+nums[j]==target)
                    {
                        n = {i,j};
                    }
                }
        }
        return n;
    }
    
};